mlir/include/mlir/Dialect/LLVMIR/LLVMAttrDefs.td

```diff
@@ -1104,4 +1104,23 @@ def TailCallKindAttr : LLVM_Attr<"TailCallKind", "tailcallkind"> {
   let assemblyFormat = "`<` $tailCallKind `>`";
 }
 
+//===----------------------------------------------------------------------===//
+// WorkgroupAttribAttr
+//===----------------------------------------------------------------------===//
+
+def WorkgroupAttribAttr
```

Comment:
Is this how that information is named in LLVM? I find "attrib" a very confusing abbreviation in a context where "attributes" are a thing.

Reply:
This is not part of LLVM, hence the `llvm.mlir.` prefix. Would it make sense to use the longer `llvm.mlir.workgroup_attribution` name?